packages/react-union/src/scan.js

```diff
@@ -69,7 +84,7 @@ const loadConfigs = (routes, descriptors) => {
  *											[{
  *												component,
  *												descriptor: {
- *													name,
+ *													widget,
```

Comment:
are you sure you want that breaking change?

Reply:
Yes, I think that at this stage of development we should encourage consistency as much as we can (keeping naming consistent across widget descriptors and the JavaScript API)

Reply:
In this case, the breaking change is negligible (and to be honest I did not even notice it before): we pass `configs` only to `onScanEnd`.